### PR TITLE
O3-989: Migrate E2e tests to o3.openmrs.org server

### PR DESCRIPTION
**Pull request by kumuditha-udayanga**
### Purpose
The purpose of this PR is to fix [O3-989](https://issues.openmrs.org/browse/O3-989).

### Approach
- Update `cypress.json`.
- Update `commands.js`

Note: This PR migrates all tests to the o3.openmrs demo server. Tests will still fail for the moment. Fixes will be sent by members assigned to each test.

### CODE REVIEW

**Review by Piumal1999 — APPROVED** [review]
Looks good to me

**Comment by kdaud** [comment]
> Note: This PR migrates all tests to the o3.openmrs demo server

@kumuditha-udayanga any reason why the tests will still fail?

**Review by kdaud — APPROVED** [review]
cc: @sherrif10

File: qaframework-bdd-tests/cypress.json [hunk 1]
```diff
@@ -6,14 +6,14 @@
   ],
   "video": true,
   "defaultCommandTimeout": 20000,
-  "baseUrl": "https://openmrs-spa.org/openmrs/spa",
+  "baseUrl": "https://o3.openmrs.org/openmrs/spa",
   "env": {
-    "API_BASE_URL": "https://openmrs-spa.org/openmrs/ws/rest/v1",
-    "ADMIN_USERNAME": "spa_user",
```
Comment (sherrif10) [inline qaframework-bdd-tests/cypress.json:12]: Meanwhile could there be a work around to resurrect this server https://openmrs-spa.org/openmrs/spa/login

**Comment by kdaud** [comment]
@Piumal1999 doesn't this finding something worth to look into before migrating the tests to use the suggested server in this commit? https://talk.openmrs.org/t/fixing-failing-3-x-e2e-tests-in-qa-dashboard/35487/3

**Review by jayasanka-sack — APPROVED** [review]
Thanks for getting this done, @kumuditha-udayanga !

File: qaframework-bdd-tests/cypress.json (hunk 1 above)
Comment (jayasanka-sack) [reply qaframework-bdd-tests/cypress.json:12]: The login is not working. Btw, are we still maintaining that server?

Comment (Piumal1999) [reply qaframework-bdd-tests/cypress.json:12]: According to this issue (https://issues.openmrs.org/projects/O3/issues/O3-955) the openmrs-spa demo will be deprecated soon and the demo will be moved to dev3/o3 servers.

**Comment by Piumal1999** [comment]
> @Piumal1999 doesn't the finding you made over the suggested server something worth to look into before migrating the tests? https://talk.openmrs.org/t/fixing-failing-3-x-e2e-tests-in-qa-dashboard/35487/3

Yeah, it's right. But, because https://dev3.opernmrs.org and https://openmrs-spa.org/openmrs/spa are not working properly, the o3 server is the only option we currently have. So I think it's ok to migrate the tests now, and I'll create the issues for the missing components soon.

**Pull request merged**